package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "./node_modules/.bin/eslint ./src/**",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "config": "yarn && ./node_modules/.bin/gulp libs"
+    "config": "yarn && ./node_modules/.bin/gulp libs && ./node_modules/.bin/gulp build"
```

Comment:
Does this run for you? The previous command doesn't work for me either unless I put the binary path in quotes, like `"yarn && \"./node_modules/.bin/gulp\" libs"`.

Reply:
![image](https://user-images.githubusercontent.com/8043941/66695196-21e4ed00-ec84-11e9-8225-c2d0444288c0.png)
It does.

Reply:
I keep forgetting and doing everything with `npm` since that's what I'm used to. With `yarn` it works fine.

Reply:
In the future I would like to support either npm or yarn for the project. For now this is fine unless you have any ideas.